### PR TITLE
bitcoin_price: use helpers

### PR DESCRIPTION
**Pull request by lasers**
We use the nice method `self.py3.request` here.

### CODE REVIEW

File: py3status/modules/bitcoin_price.py [hunk 1]
```diff
@@ -65,6 +56,7 @@ class Py3status:
     format_separator = ', '
     hide_on_error = False
     markets = 'btceUSD, btcdeEUR'
+    request_timeout = 5
```
Comment (tobes) [inline py3status/modules/bitcoin_price.py]: to me `5` feels a bit short how about `10`?

Although I'd be inclined to just remove this option as we should really do this sort of thing universally for all modules that make requests.  I'd also like to add other universal options for requests including things like proxies (although `requests` would be needed for that as they are too complex for us to be supporting ourselves.

Comment (lasers) [reply py3status/modules/bitcoin_price.py]: `10` and option now hardcoded / concealed. 👍

**Comment by lasers** [comment]
This module only have 3 placeholders (`market`, `price`, `symbol`) because it's cherrypicked. Fuck that. Let's merge this... and then I can start working on exposing everything next.

File: py3status/modules/bitcoin_price.py [hunk 2]
```diff
@@ -120,6 +110,7 @@ def post_config_hook(self):
             'YEN': '¥'
         }
         self.last_price = 0
+        self.request_timeout = 10
```
Comment (tobes) [inline py3status/modules/bitcoin_price.py]: I'm loathed to add this config parameter. I would like some more universal ones for this. Other than this I think this looks great. Currently we have to many PRs to find these nice nuggets

Comment (lasers) [reply py3status/modules/bitcoin_price.py]: At this moment, this is not a config param. I removed it earlier in first comment. We can merge this now? Maybe just give me permission to merge things for one day and I'll merge whatever I want. Heh.

I'll wait for your approval. This config parameter is trivial. I'm fine with skipping this in our documentations... and make it `10` for all... I think `bitcoin_price`'s URL is slow on fetching data too.

**Comment by ultrabug** [comment]
I'm okay with this too, thanks mate

**Pull request merged**